app/schemas/models/clan.schema.coffee

```diff
@@ -58,6 +58,11 @@ _.extend ClanSchema.properties,
     title: 'Metadata',
     description: 'Contains properties that help find autoclans'
     additionalProperties: true
+  }, {
+    networkClanId: c.stringID( title: 'Network Clan' )
+    subnetworkClanId: c.stringID( title: 'Sub-Network Clan' )
+    districtId: c.int( title: 'District Id', description: 'The unique district that the school is part of.' )
+    ncesId: c.float( title: 'NCES', description: 'The unique school NCES id. Should be a unique integer.')
```

Comment:
Thinking about this a little bit more, I might have named these either `ncesDistrictId` and `ncesSchoolId` or `districtId` and `schoolId`. Maybe favoring the former so that we can use `districtId` and `schoolId` as MongoDB ObjectIds aimed at actual records in our database, which could also include non-NCES schools, districts, and school- and district-like organizations.

Also, `ncesId` as a float seems strange compared to `districtId` as an int? Is that due to integer precision issues on the longer id or something?

Reply:
Thanks for clarifying those terms! `float` was due to precision weirdness that I missed in the initial network script. This can be fixed up when migrating to the better property name `ncesSchoolId`.

Will investigate cleaning this up as I investigate Clever PR, and will keep this PR open as documentation whilst iterating implementation.